data/101/731/177/101731177.geojson

```diff
@@ -2,166 +2,169 @@
-    "lbl:max_zoom":15.0,
-    "lbl:min_zoom":10.5,
-    "mps:latitude":47.44833,
-    "mps:longitude":-122.140591,
-    "mz:hierarchy_label":1,
-    "mz:is_current":1,
-    "mz:min_zoom":10.0,
-    "name:cat_x_preferred":[
-        "Cascade-Fairwood"
-    ],
-    "name:ceb_x_preferred":[
-        "Fairwood"
-    ],
-    "name:ces_x_preferred":[
-        "Cascade-Fairwood"
-    ],
-    "name:che_x_preferred":[
-        "\u0424\u0435\u0439\u0440\u0432\u0443\u0434"
-    ],
-    "name:cym_x_preferred":[
-        "Cascade"
-    ],
-    "name:deu_x_preferred":[
-        "Fairwood"
-    ],
-    "name:eng_x_preferred":[
-        "Fairwood"
-    ],
-    "name:eus_x_preferred":[
-        "Fairwood"
-    ],
-    "name:fas_x_preferred":[
-        "\u0641\u0627\u06cc\u0631\u0648\u0648\u062f"
-    ],
-    "name:fra_x_preferred":[
-        "Fairwood"
-    ],
-    "name:gle_x_preferred":[
-        "Fairwood"
-    ],
-    "name:hun_x_preferred":[
-        "Fairwood"
-    ],
-    "name:ita_x_preferred":[
-        "Fairwood"
-    ],
-    "name:lld_x_preferred":[
-        "Fairwood"
-    ],
-    "name:nan_x_preferred":[
-        "Cascade-Fairwood"
-    ],
-    "name:nld_x_preferred":[
-        "Cascade-Fairwood"
-    ],
-    "name:pol_x_preferred":[
-        "Fairwood"
-    ],
-    "name:por_x_preferred":[
-        "Cascade-Fairwood"
-    ],
-    "name:spa_x_preferred":[
-        "Cascade-Fairwood"
-    ],
-    "name:srp_x_preferred":[
-        "\u0424\u0435\u0440\u0432\u0443\u0434"
-    ],
-    "name:tat_x_preferred":[
-        "\u0424\u0435\u0439\u0440\u0432\u0443\u0434"
-    ],
-    "name:ukr_x_preferred":[
-        "\u0424\u0435\u0439\u0440\u0432\u0443\u0434"
-    ],
-    "name:urd_x_preferred":[
-        "\u0641\u06cc\u0626\u0631\u0648\u0688"
-    ],
-    "name:uzb_x_preferred":[
-        "Cascade-Fairwood"
-    ],
-    "name:vol_x_preferred":[
-        "Cascade-Fairwood"
-    ],
-    "qs:a0":"United States",
-    "qs:a1":"*Washington",
-    "qs:a1_lc":"53",
-    "qs:a1r":"*",
-    "qs:adm0":"United States",
-    "qs:id":0,
-    "qs:la_lc":"*",
-    "qs:level":"locality",
-    "qs:loc":"Fairwood",
-    "qs:loc_alt":"Fairwood CDP",
-    "qs:loc_lc":"5323160",
-    "qs:pop":0,
-    "qs:source":"AUS Census",
-    "qs:type":"G4210",
-    "src:geom":"quattroshapes",
-    "src:geom_alt":[],
-    "src:lbl_centroid":"mapshaper",
-    "src:population":"uscensus",
-    "src:population_date":"2022",
-    "uscensus:population":19655,
-    "uscensus:population_date":"2022",
-    "wof:belongsto":[
-        102191575,
-        85633793,
-        102086191,
-        85688623
-    ],
-    "wof:breaches":[],
-    "wof:concordances":{
-        "fips:code":"5323160",
-        "gn:id":5794097,
-        "gp:id":28340675,
-        "qs_pg:id":1113737,
-        "uscensus:geoid":"5323160",
-        "wd:id":"Q1515735",
-        "wk:page":"Fairwood, King County, Washington"
+    "edtf:cessation": "",
+    "edtf:inception": "",
+    "geom:area": 0.0014809213279999946,
```

Comment:
Nit and aside, looks like the Go tooling has all the precision instead of just 6 levels like we deliberately configured in Python